charms/katib-controller/src/charm.py

```diff
@@ -253,7 +253,10 @@ def set_pod_spec(self, event):
                                 "containerPort": self.model.config["metrics-port"],
                             },
                         ],
-                        "envConfig": {"KATIB_CORE_NAMESPACE": self.model.name},
+                        "envConfig": {
+                            "KATIB_CORE_NAMESPACE": self.model.name,
+                            "KATIB_DB_MANAGER_SERVICE_PORT": "6789",
```

Comment:
comment here to explain why it is hardcoded? maybe refer to the constant in db-manager's `charm.py`